README.md

```diff
@@ -606,7 +696,9 @@ The following is a list of enviroment variables supported by Sombra for its conf
+| DD_SERVICE_NAME                        | yes if your want to forward Sombra logs to transcend                                                                              | `customer_hosted_sombra`                                                                                                  | no     | The name for your Sombra.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| AWS_ACCESS_KEY_ID                      | yes if you're not hosting on AWS and want to utilize any AWS integrations      | N/A  | no     | The AWS access key ID of the IAM user with the STS:AssumeRole                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Even when sombra is hosted in AWS, would this not be required?

Reply:
I think AWS sets this up for you, if you run the instance there?

Reply:
I was under the impression thats the case

Reply:
> I think AWS sets this up for you, if you run the instance there?

Can we change it to `yes if you're want to utilize any for the AWS integrations`, which will be valid for any case?